aries-site/src/pages/learn/the-box-model-part-one.mdx

```diff
@@ -0,0 +1,227 @@
+import { Box, Image } from 'grommet';
+
+In this tutorial you will:
```

Comment:
In `How to add search and filter to DataTable with Data` we start off with Prerequisites should we do the same here and take out the "In this tutorial you will:"?

Reply:
I also like the idea of having the "In this tutorial, you will" that lets users know if they are on the correct page and if they want to move forward in the page so I do see the value. Maybe we can add it to the "How to add search and filter to DataTable with Data" page as well.

Reply:
Resolving this for now, we can decide on future "how-to guides" how we might want to refine.